src/ToDo/Presentation/TaskNoteViewModel.cs

```diff
@@ -2,5 +2,23 @@ namespace ToDo.Presentation;
 
 public partial class TaskNoteViewModel
 {
-    
-}
\ No newline at end of file
+	private INavigator Navigator { get; }
+
+	public ICommand AddNoteCommand { get; }
+
+	public string? Note { get; set; }
+
+	public TaskNoteViewModel(
+		INavigator navigator)
+	{
+
+		Navigator = navigator;
+
+		AddNoteCommand = new AsyncRelayCommand(AddNote);
+	}
```

Comment:
Please use the `ICommandBuilder` from the `Uno.Extensions`:

```suggestion
	public TaskNoteViewModel(
		INavigator navigator,
		ICommandBuilder addNote)
	{
		_navigator = navigator;
		
		addNote.Then(AddNote);
	}
```

Reply:
fixed